subsys/bluetooth/audio/bap_unicast_server.c

```diff
@@ -28,6 +29,26 @@ LOG_MODULE_REGISTER(bt_bap_unicast_server, CONFIG_BT_BAP_UNICAST_SERVER_LOG_LEVE
 
 static const struct bt_bap_unicast_server_cb *unicast_server_cb;
 
+int bt_bap_unicast_server_register(const struct bt_bap_unicast_server_register_param *param)
+{
+	if (param == NULL) {
+		LOG_ERR("param is NULL");
```

Comment:
```suggestion
		LOG_DBG("param is NULL");
```

Please refrain from using `LOG_ERR` in cases where we can return a value to the caller